src/components/movies/MovieList.vue

```diff
@@ -20,19 +20,31 @@ export default {
     MovieCard,
   },
   data() {
-    return {};
+    return {
+      selectedGenre: "",
+    };
   },
   props: {
     search: {
       type: String,
       default: "",
     },
   },
-
   computed: {
     searchedMovies() {
       let re = new RegExp(this.search, "i");
-      return this.moviesState.filter((movie) => movie.title.match(re));
+      return this.$store.getters.movieList.filter((movie) =>
+        movie.title.match(re)
+      );
+    },
+    moviesByCategory() {
+      const moviesByCategory = this.searchedMovies.filter(
+        (movie) => movie.genre.name === this.$store.getters.selectedGenre
+      );
+      return this.$store.getters.selectedGenre == "All Movies" ||
+        this.$store.getters.selectedGenre == ""
```

Comment:
I was wondering, if it would be easier to assign getters to separated computed values - you wouldn't have to repeat all this long 'this.$store.getters.selectedGenre'... :)

Reply:
btw. did you know that e.g. `[] == ''` is `true`? this is why in general using `==` should be avoided (always). I can't imagine the case that cannot be implemented with `===`. The second is much more intuitive and verbose.

Reply:
@rossinek That was not intentional use of ==. I think it was just a weak moment 🙈 
@KatrinDav Thank you for a nice tip❤️  I went for mapGetters istead of computed to make this shorter